Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Use the official PHP 8.1 image with FPM
+FROM php:8.1-fpm
```

Comment:
I'd like to see you using something that is PHP8.3 which is what the latest version of Magento uses.